lib/ansible/modules/cloud/openstack/os_stack.py

```diff
@@ -166,6 +172,7 @@
 def _create_stack(module, stack, cloud):
     try:
         stack = cloud.create_stack(module.params['name'],
+                                   module.params['tag'],
```

Comment:
Let's specify parameter name here (tags=) since that's a keyword parameter and will protect against additional changes to that API in shade.